### PR TITLE
Player count option for MOTD

### PR DESCRIPTION
**Pull request by moisterrific**
?????? HAVE YOU UPDATED THE CHANGELOG? ??????
yep

### CODE REVIEW

**Comment by bartico6** [comment]
I would personally split this into a few placeholders:

%onlineplayers% and %serverslots%. Maybe someone wants their MOTD to say just "There's N players online!"

**Comment by moisterrific** [comment]
> 
> 
> I would personally split this into a few placeholders:
> 
> %onlineplayers% and %serverslots%. Maybe someone wants their MOTD to say just "There's N players online!"

great idea this makes a lot more sense

**Review by bartico6 — DISMISSED** [review]
LGTM

File: TShockAPI/TSPlayer.cs [hunk 1]
```diff
@@ -1526,8 +1526,10 @@ public void SendFileTextAsMessage(string file)
 					}
 
 					foo = foo.Replace("%map%", (TShock.Config.Settings.UseServerName ? TShock.Config.Settings.ServerName : Main.worldName));
-					foo = foo.Replace("%players%", String.Join(",", players));
+					foo = foo.Replace("%players%", String.Join(", ", players));
 					foo = foo.Replace("%specifier%", TShock.Config.Settings.CommandSpecifier);
+					foo = foo.Replace("%onlineplayers%", Convert.ToString(TShock.Utils.GetActivePlayerCount()));
```
Comment (QuiCM) [inline TShockAPI/TSPlayer.cs]: Any particular reason for using `Convert.ToString(...)` over `.ToString()`?

Comment (moisterrific) [reply TShockAPI/TSPlayer.cs]: CS1503: cannot convert 'method group' to 'char'

idk

Comment (moisterrific) [reply TShockAPI/TSPlayer.cs]: nvm lol im dumb

**Review by QuiCM — APPROVED** [review]

**Pull request merged**